.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
           go-version: 1.18
       - name: Check out code
         uses: actions/checkout@v3
+      - name: Run unit tests
+        run:  go test $(go list ./...)
```

Comment:
I'd let that one be running after the linter because linting is usually faster than executing the tests.

(I already added the test invocation there)

Reply:
ups...